modin/pandas/test/dataframe/test_default.py

```diff
@@ -116,6 +116,7 @@ def test_to_numpy(data):
     assert_array_equal(modin_df.values, pandas_df.values)
 
 
+@pytest.mark.skipif(StorageFormat.get() == "Hdk", reason="Not supported by HDK")
```

Comment:
it would also be great to add the issue number for all such messages.

Reply:
There is no an open issue for this case and I'm not sure if it's required. The test uses internals that are not implemented in the HDK backend - partition.to_numpy().

Reply:
Added partition.to_numpy() implementation.